### ISSUE TITLE
Examples refer to non-static methods

### ISSUE THREAD

**Issue by pires** [bivas/protobuf-java-format#17]
`JsonFormat.printToString(...)` is not static so the examples don't work.

**Comment by pires** [comment]
The following doesn't work either. Not only `merge(...)` isn't static but it doesn't accept `String` as argument.
```
String jsonFormat = _load json document from a source_;
JsonFormat.merge(jsonFormat, builder);
```

**Comment by bivas** [comment]
Addressed by pull request #18
